File: goodjob/app/src/main/java/com/goodjob/classes/ValidSession.java
package com.goodjob.classes;

public class ValidSession {

    public static User usuarioLogueado = null;
    public static Empresa empresaLogueada = null;
//    private static final String LOCAL = "https://192.168.1.2";
    //private static final String LOCAL = "https://www.inmedia.pe/models"; // "https://anthostudio.ga";
    private static final String LOCAL = "https://goodjob-s601.000webhostapp.com";
    public static final String IP = LOCAL + "/Conexiones";
    public static final String IP_IMAGENES = LOCAL + "/imagenes";
    public static final String IMAGENES_ACTIVIDADES = IP_IMAGENES + "/imagenes_actividades/";
    public static final String IMAGENES_PRODUCTOS = IP_IMAGENES + "/imagenes_productos/";
}

File: goodjob/app/src/main/java/com/goodjob/classes/ValidSession.kt
package com.goodjob.classes

object ValidSession {

    var usuarioLogueado: User? = null
    var empresaLogueada: Empresa? = null
    private val LOCAL = "https://goodjob-s601.000webhostapp.com"
    val IP = "$LOCAL/Conexiones"
    val IP_IMAGENES = "$LOCAL/imagenes"
    val IMAGENES_ACTIVIDADES = "$IP_IMAGENES/imagenes_actividades/"
    val IMAGENES_PRODUCTOS = "$IP_IMAGENES/imagenes_productos/"
}